02_snake_animation/main.go

```diff
@@ -29,7 +34,7 @@ type snake struct {
 
 // newSnake returns a new struct instance representing a snake.
 func newSnake() snake {
-	return snake{coord{6, 6}, coord{0, -1}}
+	return snake{coord{10, 10}, coord{0, -1}}
```

Comment:
These values are declared as constants in line 19. Why wouldn't you reuse the constants?